app/javascript/components/analytics/display/WorkflowChart.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Bar, BarChart, CartesianGrid, Legend, ResponsiveContainer, Tooltip, XAxis, YAxis } from 'recharts';
+import CustomizedAxisTick from '../display/CustomizedAxisTick';
+import InfoTooltip from '../../util/InfoTooltip';
+
+class WorkflowChart extends React.Component {
+  render() {
+    return (
+      <div className="analytics-chart-borders">
+        <h5 className="text-center">
+          {this.props.title}
+          {this.props.tooltipKey && (
+            <span className="h6 text-left">
+              <InfoTooltip tooltipTextKey={this.props.tooltipKey} location="right"></InfoTooltip>
+            </span>
+          )}
+        </h5>
+        <ResponsiveContainer width="100%" height={400}>
+          <BarChart
+            width={500}
+            height={300}
+            data={this.props.data}
+            margin={{
+              top: 20,
+              right: 30,
+              left: 20,
+              bottom: 5,
+            }}>
+            <CartesianGrid strokeDasharray="3 3" />
+            <XAxis dataKey="name" interval={0} tick={<CustomizedAxisTick />} height={100} />
+            <YAxis />
+            <Tooltip />
+            <Legend />
+            <Bar dataKey="Exposure" stackId="a" fill="#557385" />
```

Comment:
Same as my previous comment. This assumes the use of the isolation workflow. For different diseases/jurisdictions, that is not a fair assumption. This should be configurable.

Reply:
this one was previously not configurable, probably due to fill. still think it should be?  We could loop over the bars but not sure how to determine the color with an unknown amount of workflows

Reply:
done: https://github.com/SaraAlert/SaraAlert/pull/993/commits/0f40f134cd1296ff028fa90890bad2bcb661f905